content/en/methods/notifications.md

```diff
@@ -67,7 +67,7 @@ min_id
 : String. Return results immediately newer than this ID
 
 limit
-: Integer. Maximum number of results to return. Defaults to 15 notifications. Max 30 notifications.
+: Integer. Maximum number of results to return. Defaults to 40 notifications. Max notifications depends on the limit param.
```

Comment:
this should be added to the changelog for 4.1, as this statement is incorrect

Reply:
Hi @trwnh , thanks for reviewing, did you mean to say that the changes are correct but needs to be added in change log only?

Reply:
Hey @trwnh could you please provide any lead? I'm afraid I don't understand.

Reply:
@ManVTish look at lines 42-49 "version history". add a line break after 4.0.0., then a line for 4.1.0 - something like

```
4.0.0 - added `admin.report` type\
4.1.0 - notification limit changed from 15 (max 30) to 40 (max 80)
```

then for the definition of `limit` on this line, remove the mention of "depends on the limit param" and change it back to 80